From: Alec Leamas <leamas.alec@gmail.com>
Date: Mon, 27 Aug 2018 06:58:59 +0200
Subject: Build: Add the USE_BUNDLED_DOCS cmake configuration

Origin: upstream, https://github.com/OpenCPN/OpenCPN/commit/180299afd735
Applied-Upstream: 5.0.0

The variable basically acts as a master switch for most bundling
alternatives. The ON alternative blocks the use of system
libraries for (as planned) wxsvg, xwcurl and unarr. If OFF,
cmake will use the system libraries if available.
---
 CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a100068..ba81181 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -201,6 +201,12 @@ ELSE(APPLE OR WIN32)
   SET(BUNDLE_GSHHS "NONE" CACHE STRING "Include GSHHS wallpaper map in package (NONE,MIN,[CRUDE],LOW,INTERMEDIATE,HIGH,FULL)")
 ENDIF(APPLE OR WIN32)
 
+IF(UNIX)
+  SET(USE_BUNDLED_LIBS "OFF" CACHE BOOL "Use bundled libraries instead of system's")
+ELSE(UNIX)
+  SET(USE_BUNDLED_LIBS "ON" CACHE BOOL "Use bundled libraries instead of system's")
+ENDIF(UNIX)
+
 IF(APPLE OR WIN32)
   IF(NOT BUNDLE_DOCS)
     SET (BUNDLE_DOCS "ON")
